components/drops/useDrops.ts

```diff
@@ -206,17 +206,41 @@ export const useDropMinimumFunds = (drop) => {
   }
 }
 
+// Convert the collection to hex
+// Construct the number - if is length 2 - pad it right with two zeros
+// If the length is 3 - pad it right with one zero and put the first digit in the end
+// If the lenght is 4 - swap the pair - aka first two letters go last
+// When you have number from step two construct the full format - 0x00${theMagic}0000
+const convertCollectionIdToMagicId = (id: string) => {
```

Comment:
Maybe Polkadot-js has a function to convert to hex?

Reply:
I'm not sure if there is a function to do this magic convert based on the comment.

Reply:
Yes use please utils from polkdadot utils if possible